compel/arch/mips/src/lib/include/uapi/asm/fpu.h

```diff
@@ -0,0 +1,144 @@
+#ifndef __CR_ASM_FPU_H__
+#define __CR_ASM_FPU_H__
+
+#include <sys/types.h>
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <compel/common/compiler.h>
+
+#define FP_MIN_ALIGN_BYTES		64
+
+#define FP_XSTATE_MAGIC1		0x46505853U
+#define FP_XSTATE_MAGIC2		0x46505845U
+#define FP_XSTATE_MAGIC2_SIZE		sizeof(FP_XSTATE_MAGIC2)
+
+#define XSTATE_FP			0x1
+#define XSTATE_SSE			0x2
+#define XSTATE_YMM			0x4
+
+#define FXSAVE_SIZE			512
+#define XSAVE_SIZE			832
+
+struct fpx_sw_bytes {
+	uint32_t			magic1;
+	uint32_t			extended_size;
+	uint64_t			xstate_bv;
+	uint32_t			xstate_size;
+	uint32_t			padding[7];
+};
+
+struct i387_fxsave_struct {
```

Comment:
This does not look like a MIPS ;-)
The entire fpu.h also seems like a stale copy from x86